helmfile/ibft2/charts/besu-node/templates/node-statefulset.yaml

```diff
@@ -0,0 +1,185 @@
+        release: {{ .Release.Name }}
+        component: {{ .Release.Name }}
+        app.kubernetes.io/name: {{ include "besu-node.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/port: "9545"
+        prometheus.io/path: "/metrics"
+    spec:
+      serviceAccountName: {{ .Release.Name }}-sa
+{{- if not .Values.bootnode.enabled }}
+      initContainers:
+        - name: init-bootnode
+          image: byrnedo/alpine-curl
+          command:
+            - "sh"
+            - "-c"
+            - "curl -X GET --connect-timeout 30 --max-time 10 --retry 6 --retry-delay 0 --retry-max-time 300 ${BOOTNODE_1_BESU_NODE_SERVICE_HOST}:8545/liveness"
+{{- end}}
+      containers:
+      - name: {{ .Release.Name }}
+        image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+        imagePullPolicy: {{ .Values.imagePullPolicy }}
+        resources:
+          requests:
+            cpu: "{{ .Values.node.resources.cpuRequest }}"
+            memory: "{{ .Values.node.resources.memRequest }}"
+          limits:
+            cpu: "{{ .Values.node.resources.cpuLimit }}"
+            memory: "{{ .Values.node.resources.memLimit }}"
+        env:
+          - name: POD_IP
+            valueFrom:
+              fieldRef:
+                fieldPath: status.podIP
+          - name: POD_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.name
+          - name: BOOTNODE1_PUBKEY
+            valueFrom:
+              secretKeyRef:
+                name: besu-bootnode-1-keys
+                key: enode.key
+          - name: BOOTNODE2_PUBKEY
+            valueFrom:
+              secretKeyRef:
+                name: besu-bootnode-2-keys
+                key: enode.key
+        volumeMounts:
+          - name: key
+            mountPath: /secrets
+            readOnly: true
+          - name: genesis
+            mountPath: /etc/genesis
+            readOnly: true
+          - name: config-toml
+            mountPath: /etc/besu
+            readOnly: true
+          - name: data
+            mountPath: /data
+        ports:
+          - name: json-rpc
+            containerPort: {{ .Values.node.rpc.port }}
+            protocol: TCP
+          - name: ws
+            containerPort: {{ .Values.node.ws.port }}
+            protocol: TCP
+          - name: graphql
+            containerPort: {{ .Values.node.graphql.port }}
+            protocol: TCP
+          - name: rlpx
+            containerPort: {{ .Values.node.p2p.port }}
+            protocol: TCP
+          - name: discovery
+            containerPort: {{ .Values.node.p2p.port }}
+            protocol: UDP
+          - name: metrics
+            containerPort: {{ .Values.node.metrics.port }}
+            protocol: TCP
+        command:
+          - /bin/sh
+          - -c
+        args:
+          - |
+            exec /opt/besu/bin/besu \
```

Comment:
How would I inject JVM runtime args (e.g. -Xmx, JFR options, etc?).

Perhaps we need a value to assign to `BESU_OPTS` or whatever that env var name is?

Reply:
Similarly, it'd be nice if I could add other args here via values for flags like `--Xsecp256k1-native-enabled=true`, etc.

Reply:
BESU_OPTS it is as an env var would do it :) we can add one with some sensible defaults as an example ?